package.json

```diff
@@ -8,6 +8,7 @@
     "db:diff": "prisma migrate diff --from-url=${DATABASE_URL} --to-migrations=./app/database/migrations --shadow-database-url=${SHADOW_DATABASE_URL} --script > rollback.sql",
     "db:prepare-migration": "prisma generate && prisma migrate dev --create-only --skip-seed",
     "db:generate-type": "prisma generate",
+    "db:deploy": "prisma migrate deploy",
```

Comment:
I would like to avoid regenerating the client at runtime, keeping that to the build process only. Running `db:deploy-migration` would introduce some drift between what the image is supposed to look like and it's current state, basically "we ship a thing, you run another."

However, totally doable if you prefer using that 👍

Reply:
No thats fine. Just needed to understand what is going on.